### DISCUSSION TITLE
Training own model

### DISCUSSION THREAD

**Discussion by Anlubi** [openai/whisper#411] [Q&A] [answered]
Hi, is it possible to train whisper with my our own dataset on our system? Or are we limited to use your models to use whisper for inference

I did not find any hints on how to train the model on my own.

**Comment by jianfch** [comment, marked as answer]
see #64

**Comment by sanchit-gandhi** [comment]
Check-out this blog for fine-tuning Whisper for multilingual ASR with Hugging Face Transformers: https://huggingface.co/blog/fine-tune-whisper

It provides a step-by-step guide to fine-tuning, right from data preparation to evaluation 🤗 There'a Google Colab so you can also run it as a notebook 😉

**Reply by Tony2055866** [reply]
How can I fine-tune only the translation task? I have the audio and the English text. What other modifications need to be made to the code?
